src/web/jcc/common/JaiaAPI.ts

```diff
@@ -160,14 +161,28 @@ export class JaiaAPI {
         return this.get(`jaia/task-packets-count`);
     }
 
-    getDepthContours(startDate?: string, endDate?: string) {
-        if (startDate && endDate) {
+    async getDepthContours(
```

Comment:
Since this function was modified and does not have a function comment yet we should add one.